README.md

```diff
@@ -28,6 +28,8 @@ If you want to add a conference, feel encouraged to create a PR which adds the c
 | [Flutter Heroes](https://flutterheroes.com/) 2024                  | 23rd of February           | [Flutter Heroes CfP](https://papers.synesthesia.it/flutter-heroes-2024/cfp)                                              | 4th of December 2023     | Turin, Italy & Online               | ???              |
 | [Flamecon 6](https://flame-engine.org/flamecon)                    | February 26th, 2024        | --                                                                                                                       | --                       | Online                              | 100 +            |
 | [Flutter Global Summit](https://events.geekle.us/flutter) 2024     | February 27th - 28th, 2024 | [Global Summit CfP](https://docs.google.com/forms/d/e/1FAIpQLScbZEiHXQRRjebkPQM87cisJdkibaD2qd3nRdMiADmP5129Ww/viewform) | ???                      | Online                              | 5000 +           |
+| [Flutter Connection](https://flutterconnection.io/) 2024          | April 24, 2024                     | [Flutter Connection CfP](https://flutterconnection.io/cfp)                        | February 10th, 2024 | Paris, France           | ???              |
```

Comment:
@AlexanderThiele this PR broke the raw formatting (the `|` are no longer aligned)

Reply:
Oops, you're right! will fix it